recipes/mo-catnip/meta.yml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mo-catnip" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/MetOffice/CATNIP/archive/1.0.tar.gz
+  sha256: 491e72214f2018dcd57e580956c97eb91b6df1cd6172b85a7e9e0da985733e54
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+
+  run:
+    - matplotlib-base >=2.2.3
+    - numpy >=1.15.4
+    - python >=3.6
+    - scipy >=1.2.1
+    - six >=1.12.0
+    - iris
+
+test:
+  imports:
+    - catnip
+
+about:
+  home: "https://github.com/MetOffice/CATNIP"
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENCE.txt
```

Comment:
Looks like this should be `LICENSE.txt` rather than `LICENCE.txt`.

Reply:
fixed

Reply:
This should be just "LICENCE" with no suffix. It refers to the name of the [file in the CATNIP repo](https://github.com/MetOffice/CATNIP/blob/master/LICENCE).